Tools/flatpak/setup-rpkgs/R/flatpakGeneratePkgsList.R

```diff
@@ -57,7 +57,7 @@ jaspModules <- Filter(isJaspModule, list.dirs(file.path(jaspDir, "Modules"), rec
 names(jaspModules) <- basename(jaspModules)
 
 # | HERE | you can add modules to exclude
-jaspModules <- jaspModules[setdiff(names(jaspModules), c("jaspProcessControl"))]
+#jaspModules <- jaspModules[setdiff(names(jaspModules), c("jaspProcessControl"))]
```

Comment:
This bit me :p

Reply:
ah sorry, that used to be a default :sweat_smile:, but good that you learned how the code works :wink: